src/solver/type_getters.jl

```diff
@@ -705,13 +709,15 @@ function get_simulation(config::AtmosConfig)
     end
     @info "init integrator: $s"
 
-    s = @timed_str begin
-        integrator = ClimaDiagnostics.IntegratorWithDiagnostics(
-            integrator,
-            scheduled_diagnostics,
-        )
+    if config.parsed_args["enable_diagnostics"]
```

Comment:
```suggestion
    if !isnothing(scheduled_diagnostics)
```

Reply:
I don't want to exercise this code, that is the point of this PR.

Reply:
I don't understand. If you add the suggestion above, this code is not exercised when there are no diagnostics

Reply:
So, if you want to not exercise this code, you can just comment out the diagnostics from your configuration